back-end/routes/create_account.js

```diff
@@ -11,17 +14,23 @@ const ecc = require('eosjs-ecc');
  * @return {AccountExtended}
  */
 module.exports = async function(req, res, next) {
+    const accountName = req.body.accountName
+    const commonName = req.body.commonName
+    const pubKey = req.body.pubKey
 
-    const accountability = new Accountability();
     accountability.login({
         name: 'yvo',
         permission: 'active',
         privKey: keyFromName('yvo').privKey,
     })
 
-    await createNewPerson(req.accountName, req.commonName, req.pubKey);
+    await createNewPerson(accountability, accountName, commonName, pubKey);
+
+    const blockchainAccount = await accountability.getAccount(accountName);
+
+    const accountExtendedObject = { ...blockchainAccount, common_name: commonName, type: AccountType.Human }
 
-    res.send();
+    res.send(accountExtendedObject);
 }
 
```

Comment:
keyFromName() needs to use the same key algorithm that is use on the FE, please modify and use "yvo" "Password123"

Reply:
`    const privKey = ecc.seedPrivate(crypto.createHash("sha256").update('yvo' + 'Password123').digest("hex"))

    accountability.login({
        name: 'yvo',
        permission: 'active',
        privKey: privKey,
    })`

like this?